src/mod_mam_odbc_async_pool_writer.erl

```diff
@@ -2,6 +2,16 @@
 %%% @author Uvarov Michael <arcusfelis@gmail.com>
 %%% @copyright (C) 2013, Uvarov Michael
 %%% @doc Collect messages and flush them into the database.
+%%%
+%%% Some backends allow ArcID can be not set (riak?).
+%%% This means, that there are no "jid => integer" mapping.
+%%%
+%%% But this module requires ArcID not be an integer.
```

Comment:
"requires ArcID **not** be an integer" but several lines below you mention that ArcID is required to be integer?

Reply:
to be an integer, typo